### PR TITLE
LimitNOFILE configurable by es_max_open_files

### PR DESCRIPTION
**Pull request by emanuelis**

### CODE REVIEW

**Comment by jakommo** [comment]
The default should be `65536` as ES 5.0 will check for this value and throw a warning otherwise. 
@emanuelis have you signed the CLA? If so, did you sue the same e-mail address for the commit?

**Comment by emanuelis** [comment]
At the time of my pull request 65536 was too small number of max open files limit.
I signed CLA now :)

**Comment by karmi** [comment]
Hi @emanuelis, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in yout Git [commit](https://github.com/elastic/ansible-elasticsearch/pull/106.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

**Comment by emanuelis** [comment]
Sure. Added one more email to my github profile.

**Comment by karmi** [comment]
@emanuelis, thanks!, as you can see the CLA check is now green and all is good.

**Comment by jakommo** [comment]
@emanuelis but you set it to `65535` which is < `65536`

**Comment by jakommo** [comment]
@gingerwizard are you ok with the merge? LGTM

**Comment by emanuelis** [comment]
Changed to 65536. I just need the ability to configure a greater value. The default doesn't matter :)

File: defaults/main.yml [hunk 1]
```diff
@@ -17,4 +17,6 @@ es_pid_dir: "/var/run/elasticsearch"
 es_data_dirs: "/var/lib/elasticsearch"
 es_log_dir: "/var/log/elasticsearch"
 es_work_dir: "/tmp/elasticsearch"
-es_plugin_dir: "/usr/share/elasticsearch/plugins"
\ No newline at end of file
+es_plugin_dir: "/usr/share/elasticsearch/plugins"
+es_max_open_files: 65536
+
```
Comment (antonbormotov) [inline defaults/main.yml]: good improvement, just small comment, please: remove extra line

**Comment by gingerwizard** [comment]
@emanuelis LGTM. Thanks for this. Merging. Apologies for the late response.

**Pull request merged**